docs/node-operators/migration-guides/upgrade-to-0701.md

```diff
@@ -0,0 +1,235 @@
+
+To load the checkpoint, find more information in the [restart network guide](../how-to/restart-network.md#load-checkpoint) 
+
+1. One of the validators will adjust [the genesis file ↗](https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json) in the [Vega Protocol networks repository ↗](https://github.com/vegaprotocol/networks). You also may ask the vega team member to review it.
+2. The person responsible for updating genesis needs to create a PR with changes.
+3. All of the validators need to accept changes and approve the PR.
+4. Approved PR must be merged by one of the validators.
+
+:::note
+Make sure, you have correct values for the following parameters in your genesis: `genesis_time`, `time_iota_ms`, `chain_id`, `network_parameters_checkpoint_overwrite`.
+:::
+
+:::note The checkpoint parameters overwrite
+All of the network parameters will be taken from checkpoint except:
+- Added since the previous released version
+- Mentioned in the `network_parameters_checkpoint_overwrite` section
+:::
+
+### 6. Download new genesis file
+After creating a backup and preparing a new genesis file, put it on your server. All the validators **must** use the same genesis file. 
+
+1. Download genesis
+2. Remove old genesis at `<TENDERMINT-HOME>/config/genesis.json`
+3. Save new, downloaded genesis to `<TENDERMINT-HOME>/config/genesis.json`
+4. Verify genesis - see example below
+
+#### Example workflow
+An example workflow for reviewing the genesis file may look like following:
+
+```bash 
+# Download genesis
+wget https://raw.githubusercontent.com/vegaprotocol/networks/master/mainnet1/genesis.json
+
+# Move old genesis to a different location
+cp <TENDERMINT-HOME>/config/genesis.json <TENDERMINT-HOME>/config/genesis.json.bk
+
+# Copy genesis to its final location
+cp ./genesis.json <TENDERMINT-HOME>/config/genesis.json
+
+# Verify genesis
+<VEGA-BIN> verify genesis <TENDERMINT-HOME>/config/genesis.json
+```
+
+### 7. Update Vega core config
+
+There is no change required in the `vega core` config. If you have to prepare the `vega` config from scratch, use the instruction for (v0.68.2)[(./upgrade-to-0682.md#7-update-vega-core-config)]
+
+### 8. Update Tendermint config
+
+There is no change required in the `tendermint` config. If you have to prepare the `tendermint` config from scratch, use the instruction for (v0.68.2)[(./upgrade-to-0682.md#8-update-tendermint-config)]
+
+### 9. Update data-node config
+
+If you were running node in version `v0.68.0+`, the only think you have to update is the `chain_id` in the `data-node` config. If you need to setup data node from scratch, please follow documentation for [v0.68.0](./upgrade-to-0682.md#9-update-data-node-config)
+
+:::warning Update `chain_id`
+It is important to update the chain_id for your data node config, otherwise your data node will fail.
+:::

```

Comment:
```suggestion
### 9. Update data node config

If you were running a data node in version `v0.68.0+`, the only thing you have to update is the `chain_id` in the `data-node` config. If you need to set up data node from scratch, please follow documentation for [v0.68.0](./upgrade-to-0682.md#9-update-data-node-config)

:::caution Update `chain_id`
It is important to update the chain ID for your data node config, otherwise your data node will fail.
:::

```